.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["next/core-web-vitals", "plugin:react/recommended"],
+  "extends": ["next/core-web-vitals", "plugin:react/recommended", "prettier"],
```

Comment:
I added Prettier to the project. Please pull my changes and after that run `npm install`.

You might also want to install the Prettier extension for VS Code if that's the editor that you're using.

In VS Code run the `Show recommended extensions` command and you'll see the option to install the Prettier extension.

Reply:
I have one but somethings wrong in my vscode when I open this project, its terminating the host the error said